discovery-infra/test_infra/utils.py

```diff
@@ -373,7 +373,11 @@ def wait_till_cluster_is_in_status(
 def is_cluster_in_status(client, cluster_id, statuses):
     log.info("Is cluster %s in status %s", cluster_id, statuses)
     try:
-        return client.cluster_get(cluster_id).status in statuses
+        if client.cluster_get(cluster_id).status in statuses:
+            return True
+        else:
+            log.info(f"Cluster not yet in its required status. "
+                     f"Current status: {client.cluster_get(cluster_id).status}")
```

Comment:
why not to get cluster once and not to work with object? And lets return false from this func and not None

Reply:
Fixed